executors/injectexec/hooks_test.go

```diff
@@ -0,0 +1,191 @@
+//
+// Copyright (c) 2018 The heketi Authors
+//
+// This file is licensed to you under your choice of the GNU Lesser
+// General Public License, version 3 or any later version (LGPLv3 or
+// later), or the GNU General Public License, version 2 (GPLv2), in all
+// cases as published by the Free Software Foundation.
+//
+
+package injectexec
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/heketi/tests"
+
```

Comment:
this new line can be removed

Reply:
I am trying to group imports by origin, first by std library, then by external dependency, then by internal (in-repo) package. The "heketi/tests" package is not part of "heketi/heketi" and that's why its by itself in a single line group in some of these.